contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.18;
+
+import "../../math/SafeMath.sol";
+import "../Crowdsale.sol";
+
+
+/**
+ * @title TimedCrowdsale
+ * @dev Crowdsale accepting contributions only within a time frame.
+ */
+contract TimedCrowdsale is Crowdsale {
+  using SafeMath for uint256;
+
+  uint256 public startTime;
+  uint256 public endTime;
+
+  /**
+   * @dev Reverts if not in crowdsale time range. 
+   */
+  modifier inTimeRange {
```

Comment:
(i) perhaps it's worth using the `only*` prefix here?

(ii) I feel like this could be named more crowdsale-specific like `onlyWhileOpen` or `whileCrowdsaleOpen` or `timeIsValid`

Reply:
I like using `only`

Reply:
Thanks, changed to `onlyWhileOpen`, and, since we are on it, changed `hasExpired` to `hasClosed`, to be consistent with the _open_/_closed_ idea for `TimedCrowdsale` (note that it is still different from _ended_, which can happen prematurely.. Maybe ideally we would want `openingTime` and `closingTime`?)